aws/v2/auth.go

```diff
@@ -0,0 +1,35 @@
+package aws
```

Comment:
NIT: this will be annoying to `import` alongside the real `aws` package. Doubly so in Terratest and other repos that also have their _own_ `aws` packages too.

Reply:
Makes sense. What are your thoughts on renaming this to `awscommons`?

Reply:
bias for action: went forward with the rename - 0e40a65

Reply:
Works for me, thx!